ext/console/02_console.js

```diff
@@ -783,10 +749,14 @@
     switch (typeof value) {
       case "string": {
         if (value.length > STR_ABBREVIATE_SIZE) {
-          const trunc = multibyteAwareTruncate(value, STR_ABBREVIATE_SIZE - 3);
-          if (value !== trunc) {
-            value = trunc + "...";
+          value = value.substring(0, STR_ABBREVIATE_SIZE - 3);
+          // If the last byte is the leading byte of a multi byte character,
+          // remove that character.
+          const lastChar = value.charCodeAt(value.length - 1);
+          if (lastChar >= 0xD800 && lastChar <= 0xDBFF) {
+            value = value.substring(0, value.length - 1);
```

Comment:
You could use `value.at(-1)` and do a string comparison. Similarly with `value.slice(0, -1)`.

Also, these operations should use primordials.